lib/spack/spack/util/executable.py

```diff
@@ -226,15 +226,16 @@ def streamify(arg, mode):
                     "If multiple levels of quotation are required, use " "`ignore_quotes=True`.",
                 )
 
-        cmd = self.exe + list(args)
+        cmds = self.exe + list(args)
+        cmd = ['"' + c + '"' if "import" in c else c for c in cmds]
 
         escaped_cmd = ["'%s'" % arg.replace("'", "'\"'\"'") for arg in cmd]
         cmd_line_string = " ".join(escaped_cmd)
         tty.debug(cmd_line_string)
 
         try:
             proc = subprocess.Popen(
-                cmd, stdin=istream, stderr=estream, stdout=ostream, env=env, close_fds=False
```

Comment:
I think this is going to cause issues, but I need to think a bit more about how we can rework this for this use case and to preserve previous behavior.

Reply:
I decided to keep the list comprehensions for now as I thought its more succinct